modules/cluster-logging-deploy-es-cli.adoc

```diff
@@ -0,0 +1,145 @@
+// Module included in the following assemblies:
+//
+// * logging/log_storage/installing-log-storage.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="cluster-logging-deploy-es-cli_{context}"]
+= Installing the {es-op} by using the CLI
+
+You can use the {oc-first} to install the {es-op}.
+
+.Prerequisites
+
+* Ensure that you have the necessary persistent storage for Elasticsearch. Note that each Elasticsearch node requires its own storage volume.
++
+[NOTE]
+====
+If you use a local volume for persistent storage, do not use a raw block volume, which is described with `volumeMode: block` in the `LocalVolume` object. Elasticsearch cannot use raw block volumes.
+====
++
+Elasticsearch is a memory-intensive application. By default, {product-title} installs three Elasticsearch nodes with memory requests and limits of 16 GB. This initial set of three {product-title} nodes might not have enough memory to run Elasticsearch within your cluster. If you experience memory issues that are related to Elasticsearch, add more Elasticsearch nodes to your cluster rather than increasing the memory on existing nodes.
+
+ifdef::openshift-origin[]
+* Ensure that you have downloaded the {cluster-manager-url-pull} as shown in "Obtaining the installation program" in the installation documentation for your platform.
++
+If you have the pull secret, add the `redhat-operators` catalog to the OperatorHub custom resource (CR) as shown in *Configuring {product-title} to use Red Hat Operators*.
+endif::[]
+
+* You have administrator permissions.
+* You have installed the {oc-first}.
+
+.Procedure
+
+. Create a `Namespace` object as a YAML file:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: openshift-operators-redhat <1>
+  annotations:
+    openshift.io/node-selector: ""
+  labels:
+    openshift.io/cluster-monitoring: "true" <2>
+----
+<1> You must specify the `openshift-operators-redhat` namespace. To prevent possible conflicts with metrics, you should configure the Prometheus Cluster Monitoring stack to scrape metrics from the `openshift-operators-redhat` namespace and not the `openshift-operators` namespace. The `openshift-operators` namespace might contain community Operators, which are untrusted and could publish a metric with the same name as
```

Comment:
```suggestion
<1> You must specify the `openshift-operators-redhat` namespace. To prevent possible conflicts with metrics, consider configuring the Prometheus Cluster Monitoring stack to scrape metrics from the `openshift-operators-redhat` namespace and not the `openshift-operators` namespace. The `openshift-operators` namespace might contain community Operators, which are untrusted and could publish a metric with the same name as
```
![Screenshot from 2023-12-18 15-21-55](https://github.com/openshift/openshift-docs/assets/57954076/b0ecd922-016f-4a53-a554-e2bc86106b42)